src/servers/main.cc

```diff
@@ -462,8 +462,9 @@ std::vector<Option> options_
       {OPTION_TRACE_FILEPATH, "trace-file", Option::ArgStr,
        "Set the file where trace output will be saved."},
       {OPTION_TRACE_LEVEL, "trace-level", Option::ArgStr,
-       "Set the trace level. OFF to disable tracing, MIN for minimal tracing, "
-       "MAX for maximal tracing. Default is OFF."},
+       "Set the trace level. OFF to disable tracing, TIMESTAMPES to trace "
```

Comment:
We need to allow --trace-level to be specified more than once because there are multiple trace levels that can be enabled.  For example, see how --backend-config is handled. TIMESTAMPS enabled only timestamps and TENSORS enables only tensors.